ginivision/src/main/java/net/gini/android/vision/returnassistant/LineItemsAdapter.kt

```diff
@@ -134,14 +131,18 @@ sealed class ViewHolder<in T>(itemView: View, val viewType: ViewType) :
     }
 
     class LineItemViewHolder(itemView: View) : ViewHolder<SelectableLineItem>(itemView, LineItem) {
+        private val card: MaterialCardView = itemView.gv_line_item
         private val checkbox: CheckBox = itemView.gv_checkbox
         private val description: TextView = itemView.gv_description
         private val quantity: TextView = itemView.gv_quantity
+        private val quantityLabel: TextView = itemView.gv_quantity_label
+        private val edit: Button = itemView.gv_edit
         private val priceIntegralPart: TextView = itemView.gv_amount_integral_part
         private val priceFractionPart: TextView = itemView.gv_amount_fraction_part
         var listener: LineItemsAdapterListener? = null
 
         override fun bind(data: SelectableLineItem, allData: List<SelectableLineItem>?) {
+            if (data.selected) enable() else disable()
```

Comment:
Naughty to drop the {}s. It's a magnet for bugs.

Reply:
You're right 🙈